incapsula/resource_site.go

```diff
@@ -60,6 +60,51 @@ func resourceSite() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
+			"site_id": &schema.Schema{
```

Comment:
I am the one that removed `site_id` curious why you want to put it back when `d.Id()` is the `site_id`

Reply:
Ah I see it below being used home the other comment makes more sense.